setup.py

```diff
@@ -328,6 +328,21 @@ def run(self):
             # gets built
             ninja_global.run()
 
+        def copy_file(src, dst, *args, **kwargs):
+            print('copying {} -> {}'.format(src, dst))
+            return shutil.copy2(src, dst, *args, **kwargs)
+
+        def copy_tree_if_not_exists(src, dst):
+            if os.path.exists(dst):
+                print('{} already exists, skipping'.format(dst))
+            else:
+                shutil.copytree(src, dst, copy_function=copy_file)
+
+        copy_tree_if_not_exists('torch/csrc', 'torch/lib/include/torch/csrc/')
+        copy_tree_if_not_exists('torch/lib/pybind11/include/pybind11/',
```

Comment:
not exists might not be enough - if the headers have changed we won't update them

Reply:
Is it ok to just delete the directories first and then always copy them?